load-agent/locustClient.py

```diff
@@ -261,6 +261,14 @@ def issuer_getinvite(self):
     def issuer_getliveness(self):
         return self.issuer.is_up()
 
+    @stopwatch
+    def delete_oob(self, id):
+        self.run_command({"cmd": "deleteOobRecordById", "id": id})
+
+        line = self.readjsonline()
+
+        return ""
```

Comment:
I don't think we need to return anything 

Also, just for a sanity check, has this been tested / shown to work? - Do we need the 
```line = self.readjsonline()```?

Reply:
Yeah that `line` in `line = self.readjsonline()` is unused, but I'd seen that in a few other methods here as well (where `line` isn't used either, `ping_mediator`, `receive_credential`, `presentation_exchange` for example) so left it in.

Then if I take it out I started getting errors like 
```
load-agent-1  | [2024-03-21 16:56:13,560] load-agent/ERROR/locust.user.task: 'NoneType' object is not subscriptable
load-agent-1  | Traceback (most recent call last):
load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 347, in run
load-agent-1  |     self.execute_next_task()
load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 372, in execute_next_task
load-agent-1  |     self.execute_task(self._task_queue.pop(0))
load-agent-1  |   File "/usr/local/lib/python3.8/dist-packages/locust/user/task.py", line 384, in execute_task
load-agent-1  |     task(self)
load-agent-1  |   File "/load-agent/load-agent/locustMediatorIasIssueDeleteOob.py", line 62, in delete_oob_record
load-agent-1  |     self.client.delete_oob(self.connection["oobRecordId"])
load-agent-1  | TypeError: 'NoneType' object is not subscriptable
```
hadn't dug into the cause so left it in like the other methods.

The `return` is indeed unneeded so I'll take that out.